chart/tests/deployment_test.yaml

```diff
@@ -108,3 +108,12 @@ tests:
   - contains:
       path: spec.template.spec.containers[0].args
       content: "--no-cacerts"
+- it: should add CATTLE_SYSTEM_CATALOG to env
```

Comment:
What is this field? A description in helm?

Reply:
this is https://github.com/lrills/helm-unittest used by rancher chart.

Reply:
This is a test that proves the template is rendered correctly when the value is set. 

@guangbochen do we need a test to prove that it is not present when unset?

Reply:
The not present case should already be covered in https://github.com/rancher/rancher/blob/6e0c387a97acf46307eb57f941bcdddcd6d9635c/chart/tests/deployment_test.yaml#L44